docs/userguide/appendices.rest

```diff
@@ -2611,16 +2845,16 @@ A17.2. Land surface diagnostic output variables
       :file: output-tables/LSMOUT.csv
       :header-rows: 1
 
-A17.3. Streamflow output variables at all channel reaches/cells
+A19.3. Streamflow output variables at all channel reaches/cells
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
    :file:`{YYYYMMDDHHMM}.CHRTOUT_DOMAIN{X}`
-
-   .. csv-table:: CHRTOUT
+_
+   .. csv-tAble:: CHRTOUT
```

Comment:
The `_` produces the warning `docs/userguide/appendices.rest:2852: WARNING: Block quote ends without a blank line; unexpected unindent.`

Removing it will fix the warning and using a lowercase `a` in `csv-table` will make things more uniform.

Reply:
That's odd - not sure how that got messed up! Thanks for catching that.